frontend-react/src/AppRouter.tsx

```diff
@@ -242,6 +244,30 @@ export const appRoutes: RouteObject[] = [
                         path: "facilities-providers",
                         element: <FacilitiesProvidersWithAuth />,
                     },
+                    {
+                        path: "facility-provider-submitter-details/facility/:reportId",
+                        element: (
+                            <FacilityProviderSubmitterDetailsWithAuth
+                                aggregatorType={AggregatorType.FACILITY}
+                            />
+                        ),
+                    },
+                    {
+                        path: "facility-provider-submitter-details/provider/:reportId",
+                        element: (
+                            <FacilityProviderSubmitterDetailsWithAuth
+                                aggregatorType={AggregatorType.PROVIDER}
+                            />
+                        ),
+                    },
+                    {
+                        path: "facility-provider-submitter-details/submitter/:reportId",
+                        element: (
+                            <FacilityProviderSubmitterDetailsWithAuth
+                                aggregatorType={AggregatorType.SUBMITTER}
+                            />
+                        ),
+                    },
```

Comment:
Should we rethink the url structure? Should this be `reports/:reportId/facility`, `reports/:reportId/facility/provider` and `reports/:reportId/facility/submitter`? Also, in either case, these subpaths should be children of their parent folder (ex: path facility-provider-submitter-details, children: facility, provider, submitter)

Reply:
@jpandersen87 this should have been aggregatorId. Updated.

Reply:
Will change to contain children